filemanager/static/filemanager/css/style.css

```diff
@@ -1,43 +1,50 @@
-*{
+* {
```

Comment:
It makes it easier for humans to parse, and for people who care about an extra byte before compressing, it's not hard to minify with something like rcssmin...